sojourn-station.dme

```diff
@@ -2886,6 +2886,7 @@
 #include "code\modules\projectiles\guns\projectile\automatic\triage.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\trouble_shooter.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\vector.dm"
+#include "code\modules\projectiles\guns\projectile\boltgun\baroque.dm"
```

Comment:
Out of order!
make sure to open this in dm and hit complie to fix

Reply:
Sorry, what do you mean by "dm"?

Reply:
Nevermind, realized you meant dreammaker, working on it now.

Reply:
dme has been compiled and re-uploaded, try it now.